modules/apps/frontend-js/frontend-js-minifier/src/main/java/com/liferay/frontend/js/minifier/internal/GoogleJavaScriptMinifier.java

```diff
@@ -72,7 +73,13 @@ public String compress(String resourceName, String content) {
 				SourceFile.fromCode("extern", StringPool.BLANK), sourceFile,
 				compilerOptions);
 
-			if (compiler.hasErrors()) {
+			boolean ignoredFile = Stream.of(
+				_IGNORED_FILES
+			).anyMatch(
+				resourceName::endsWith
+			);
+
+			if (compiler.hasErrors() || ignoredFile) {
```

Comment:
I would check for `ignoredFile` right before the minification takes place. That way you skip a step in the process which would only consume time for nothing.

I assume what you wanted to do here was to let the minification happen on ignored files if it was successful, but I think it is better to err on minifiying less than to err on minifying more. Also, in any case, I plan to remove the list and find some way to skip minification for all known portal resources in the future, so we may take this as a temporary fix.

Reply:
It makes sense to me. Just updated.